src/folio_migration_tools/marc_rules_transformation/rules_mapper_base.py

```diff
@@ -4,6 +4,7 @@
 import time
 import uuid
 from textwrap import wrap
+from folio_migration_tools.folder_structure import FolderStructure
```

Comment:
I think this is a duplicate import statement (see line 21) and can be removed. The second import is located within the context of the other imports.

Reply:
Fixed lints